client/my-sites/marketing/buttons/tray.jsx

```diff
@@ -161,11 +162,21 @@ class SharingButtonsTray extends React.Component {
 	getLimitedButtonsNoticeElement = () => {
 		if ( this.props.limited ) {
 			return (
-				<em className="sharing-buttons-preview__panel-notice">
-					{ this.props.translate( 'Sharing options are limited on private sites.', {
-						context: 'Sharing: Buttons',
-					} ) }
-				</em>
+				<p className="sharing-buttons-preview__panel-notice">
+					<em>
+						{ this.props.translate(
+							'It looks like your site is Private. To get more sharing options make your site Public.',
+							{
+								context: 'Sharing: Buttons',
+							}
+						) }
+					</em>
+					<a href="https://wordpress.com/support/settings/privacy-settings/">
```

Comment:
I would personally prefer if the link opens in a new tab here. In a similar way as we have it with other links, e.g.:

![Markup on 2021-08-10 at 12:30:01](https://user-images.githubusercontent.com/25105483/128851938-d2b26ef9-83f8-4c79-a98a-44f0c51a6eb2.png)

Also, the link could include that "external link icon" to make it clear it will be opened in a new tab:

![Markup on 2021-08-10 at 12:32:01](https://user-images.githubusercontent.com/25105483/128852155-47401543-a5c1-4adb-ac1d-10a06811250e.png)